packages/forms/src/templates/ArrayFieldTemplate.js

```diff
@@ -17,7 +21,7 @@ function FieldTemplate({ element, cantDelete }) {
 					name={`btn-move-element-up-${element.index}`}
 					disabled={!element.hasMoveUp}
 					onClick={element.onReorderClick(element.index, element.index - 1)}
-					title="Move Up"
+					title={t('FIELD_TEMPLATE_ACTION_MOVE_UP', { defaultValue: 'Move Up' })}
```

Comment:
On those 3 translated keys, you should prefix it with `ARRAY_FIELD_TEMPLATE` instead of `FIELD_TEMPLATE`.
This FieldTemplate component exists only for readability, but all the file is part of the array template.